Gemfile

```diff
@@ -25,3 +25,5 @@ gem 'rspec-puppet',                                              '~> 5.0'
 gem 'rubocop',                                                   '~> 1.75'
 gem 'unf_ext',                                                   '~> 0.0.9'
 gem 'yamllint',                                                  '~> 0.0.9'
+gem 'rexml'
+gem 'bigdecimal',                                                '= 3.2.1'
```

Comment:
Can you add a comment here about why this is pinned so precisely, so we can see it and know what to check to repro the issue if we want to see if things have been fixed upstream down the line?

Reply:
Sorry, missed this.

The various gem dependencies aren't currently met unless we add this specific gem at this version.  We can revisit it as part of an overarching review of the structure.

If we don't include it, `onceover` (especially with `--parallel`) fails hard.